packages/edit-post/src/components/header/template-title/delete-template.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { pickBy } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
```

Comment:
```suggestion
import { __, sprintf } from '@wordpress/i18n';
```
Add sprintf for the template name in the confirmation.